var/spack/repos/builtin/packages/r-pegas/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RPegas(RPackage):
+    """Functions for reading, writing, plotting, analysing, and
+    manipulating allelic and haplotypic data, including from VCF files,
+    and for the analysis of population nucleotide sequences and
+    micro-satellites including coalescent analyses, linkage
+    disequilibrium, population structure (Fst, Amova) and equilibrium
+    (HWE), haplotype networks, minimum spanning tree and network, and
+    median-joining networks."""
+
+    homepage = "https://cran.r-project.org/web/packages/pegas/index.html"
```

Comment:
The homepage should be the URL, if one exists, in the DESCRIPTION file. In this case:
http://ape-package.ird.fr/pegas.html